### ISSUE TITLE
editable true in column is not working

### ISSUE THREAD

**Issue by MAfzalKhan1997** [adazzle/react-data-grid#2266]
How to make cells editable with default editor not custom. I'm on latest version 7.0.0-canary.33 and stuck don't know If I missed something. Also let me know if there's any proper docs and examples for latest version other than storybook demos.

**Comment by SO-Eng** [comment]
Hi @MAfzalKhan1997 are you using editor2 from an older version? I figured out, that editor is replaced with editor two in canary.30 I guess. So I updated from .26 to .30 and had same issue like you have.
I changed from editor 2 to editor and have same behavior like in editor2 now.

And serously that is worse. My Popper (Material UI) is now "behind" the Rows and also AnchorEl is not set correct.

![image](https://user-images.githubusercontent.com/45681404/104738054-524bee80-5745-11eb-9998-bc8f5ca148fb.png)

It should be visible like in canary.26 with editor like this:
![image](https://user-images.githubusercontent.com/45681404/104738496-e28a3380-5745-11eb-9788-85820bc532ec.png)

This needs a fix. Logic with p.onRowChange from editor2 with style from editor in canary.26
